src/Controls/tests/Core.UnitTests/Controls.Core.UnitTests.csproj

```diff
@@ -43,4 +42,12 @@
     <Compile Include="..\..\..\Core\tests\UnitTests\TestClasses\DispatcherStub.cs" Link="TestClasses\DispatcherStub.cs" />
   </ItemGroup>
 
+  <!--<ItemGroup>
+    <EmbeddedResource Include="Xaml\AppResources.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </EmbeddedResource>
+  </ItemGroup>-->
+
+  <Import Project="$(MauiSrcDirectory)Maui.InTree.props" Condition=" '$(UseMaui)' != 'true' " />
```

Comment:
I added this in order to make this compile https://github.com/dotnet/maui/pull/13660/files#diff-31ccb9fbdf77934e19c9fd281ea508f804f276e5acea61f02ca0540fef3ec0bdR72. 

We had a lot of stuff in `ControlGallery` that the `Xaml.UnitTests` was using to test referenced assemblies. It might be more accurate to just create a specific reference assembly for `Xaml.UnitTests` to use instead of trying to make `Core.UnitTests` do too many things.

Reply:
+1 on this!

Reply:
Will that simplify things in this PR?

Reply:
Also wondering if you just create a specific assembly/project for this test instead of pulling form somewhere else. So like a Controls.Core.UnitTests.ExternalAssembly.dll or something. So we can write tests for this. But, if this is for Xaml, maybe we move this test to the XAML side and add a separate assembly there. I think right now moving it into this project makes the test invalid.